.github/workflows/test.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
   push:
     branches: [master]
+  schedule:
+    - cron: "42 7 * * *"
```

Comment:
It's 09:42 AM UTC, right? My thinking here is that it's night in the US so the workers should be somewhat idle.

Reply:
Better make it 9:42, I think.